test/integration/main.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```

Comment:
Why did you replace run_integration_tests with this? The previous utility did more. For example, I could locally run all integration tests with a single, easy command and go get a drink. With this it would be a lot harden

Reply:
added an `all` test case